### DISCUSSION TITLE
Webcam Putting Full Swing Kit

### DISCUSSION THREAD

**Discussion by ryanchiro7** [springbok/MLM2PRO-GSPro-Connector#139] [General]
I cannot get connected to GSPro. Seems the camera is connecting. I've watched every video and read everything I can find. I've spent hours trying to get it to work with no success. If someone could reach out to me with some help, I would really appreciate it. Thank you.
Might be easier to communicate over FB Messeger: Parker Ryan https://www.facebook.com/share/19SzPPRpHn/?mibextid=kFxxJD
![IMG_8389](https://github.com/user-attachments/assets/308e8213-846d-431a-a982-922751b0bf79)
![IMG_8390](https://github.com/user-attachments/assets/4fca125e-5759-4084-afed-f42f1f1fca2a)
![IMG_8391](https://github.com/user-attachments/assets/135c1902-a1a5-4ff4-baf6-c1622fc063d4)

**Comment by wonder99** [comment]
Which time zone are you in?

**Reply by ryanchiro7** [reply]
MST

**Comment by Amwldinho** [comment]
did you solve it? I have the same issue and cannot connect to gspro

**Reply by springbok** [reply]
Can you upload your log file, make sure there aren't any windows security settings blocking the port. Also make sure you are using the OpenAPI GSPro license.

**Reply by Eday1515** [reply]
I am having the same issue. Was there a fix for this. I don’t know if you can connect through open api? If so how do I do that.

**Comment by MyswingSux** [comment]
When I click start on the webcam I am not getting the putting window to open to get to the advanced settings. Anyone out there to help me? Thank you.

**Comment by 5jfzkdyst8-commits** [comment]
I have been able to get the connector to work and putt with my full swing. My issues are when I  change my club selection in gspro it does not change on the full swing and I don’t think it should since I am just mirroring my tablet anyway. But when I change my club in the full swing app on my tablet it registers a shot in gspro. Also I can not get the full swing to register anything inside 40 yards. It works when I am not using the mlm2 connector but not when I am using it. I absolutely love being able to putt and I think the putting is really accurate. Any help with these issues would be greatly appreciated.

**Reply by gimlee168-rgb** [reply]
How did you get your connection to even work? I am pretty stuck. Thanks in advance.

**Comment by gimlee168-rgb** [comment]
Has anyone gotten around this issue yet?[
<img width="820" height="479" alt="IMG_7487_JPG" src="https://github.com/user-attachments/assets/901ca04a-8c7d-4611-805d-853ad8355548" />
<img width="904" height="514" alt="IMG_7488_JPG" src="https://github.com/user-attachments/assets/6e5bb09e-0421-4041-8e52-7b718cd01d5f" />
](url)

**Reply by springbok** [reply]
> Has anyone gotten around this issue yet?

This error means that the connector could not find the gspro openapi window, you have to use the gspro openapi license for the window to appear, make sure the title of the window matches what is set in settings.

**Reply by gimlee168-rgb** [reply]
@springbok  Thanks for the response!  Do you know if Full Swing chipping will still work if I move to the open API? And that means I will need a physical phone or table to airplay to the pc right?

**Reply by springbok** [reply]
Yes you will need to mirror the app onto the PC, then setup the data points you want the connector to extract and send to gspro. I don't know much about full swing but if chipping is the same in the full swing app as other shots then it should be able to handle it. If the screen changes in the FS app when chipping then you will have problems.
